src/main/java/com/CiobanuVladimir/classesandobjects/autoservicetask/ManageAutoService.java

```diff
@@ -0,0 +1,30 @@
+package com.CiobanuVladimir.classesandobjects.autoservicetask;
+
+public class ManageAutoService {
+    public static void main(String[] args) {
+        // Declararea si initializarea obiectelor folosind toatte tipurile de constructori
+        Car mercedesBenzClasaE = new Car("E Class", "Mercedes", "Verde");
+
+        mercedesBenzClasaE.plateNumber = "PRA 985";
+        mercedesBenzClasaE.kmWhenEnteredTheService = 154478;
+
+        System.out.println("Masina " + mercedesBenzClasaE.make + " de model " + mercedesBenzClasaE.model + " de culoare" +
+                " " + mercedesBenzClasaE.color + " are numarul de inmatriculare " + mercedesBenzClasaE.plateNumber + "si" +
+                " kilometrajul " + mercedesBenzClasaE.kmWhenEnteredTheService);
+
+        Car vwPolo = new Car("HTE 455");
+        Car volvoXC90 = new Car("BOS 356");
+
+
+        Tool ciocan = new Tool("Ciocan");
+        ciocan.weight = 1.65;
+
+        System.out.printf("Instrumentul cu numele " + ciocan.name + " are greutatea de " + ciocan.weight);
```

Comment:
From what I know, printf() method is used for string formating, therefore I believe that in this case we should use println() instead of printf();